setup.py

```diff
@@ -131,7 +131,7 @@ def get_version():
   + get_ambari_server_stack_package()
   + get_extra_common_packages(),
   package_dir=create_package_dir_map(),
-  install_requires=["coilmq==1.0.1"],
+  install_requires=["coilmq==1.0.1", "cryptography==46.0.3"],
```

Comment:
I’m uncertain whether this change is necessary. If it isn’t, I can revert it, or we can relocate it to a more suitable location if needed.